### PR TITLE
Move benchmarks/dynamo/huggingface configuration to YAML 

### PR DESCRIPTION
**Pull request by kit1980**
Similar to https://github.com/pytorch/pytorch/pull/120299

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames

### CODE REVIEW

File: benchmarks/dynamo/huggingface.py [hunk 1]
```diff
@@ -104,115 +108,36 @@ def process_hf_reformer_output(out):
...
-    "YituTechConvBert": 2,
-}
-
-SKIP_ACCURACY_CHECK_MODELS = {
-    # Models too large to have eager, dynamo and fp64_numbers simultaneosuly
-    # even for 40 GB machine.
-    "DebertaV2ForMaskedLM",
-    "BlenderbotForCausalLM",
-}
+# TODO(kit1980): deduplicate with the same in torchbench.py
```
Comment (ysiraichi) [inline benchmarks/dynamo/huggingface.py:111]: What do you think about deduping this in this PR?

File: benchmarks/dynamo/huggingface.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      # even for 40 GB machine.
+      - DebertaV2ForMaskedLM
+      - BlenderbotForCausalLM
+
+only_inference:
+  # Fails with dynamo for train mode
+  - M2M100ForConditionalGeneration
+
+only_fp32:
+  - GoogleFnet
```
Comment (ysiraichi) [inline benchmarks/dynamo/huggingface.yaml:110]: These feel like other names for skipping.

**Review by ysiraichi — COMMENTED** [review]
Overall, looks good to me. At this point, I'm wondering whether we should have some sort of standard pattern for these YAML files, i.e. having a specific hierarchy.

File: benchmarks/dynamo/huggingface.py (hunk 1 above)
Comment (kit1980) [reply benchmarks/dynamo/huggingface.py:111]: I want to merge this, then do timm_models.py, and then do refactoring to remove duplications based on all three.

File: benchmarks/dynamo/huggingface.yaml (hunk 2 above)
Comment (kit1980) [reply benchmarks/dynamo/huggingface.yaml:110]: Yeah, I'm not too sure about this one. Maybe after converting this and timm_models.py, we can come up with a specific hierarchy like you said, and enforce it with a linter or test.

File: benchmarks/dynamo/huggingface.py (hunk 1 above)
Comment (shunting314) [reply benchmarks/dynamo/huggingface.py:111]: Can this be moved to benchmark/dynamo/common.py in a base PR and let the current PR stacked on that (or you can do both in the same PR)? End result is the same but I feel more comfortable with the proposed workflow

Comment (kit1980) [reply benchmarks/dynamo/huggingface.py:111]: Please see https://github.com/pytorch/pytorch/pull/131924

**Review by shunting314 — APPROVED** [review]
Please rebase upon https://github.com/pytorch/pytorch/pull/131924 to take advantage of the yaml loading utility added to common.py

**Pull request closed without merging**